test/doctool/test-apilinks.js

```diff
@@ -2,23 +2,36 @@
 
 require('../common');
 const fixtures = require('../common/fixtures');
+const fs = require('fs');
 const assert = require('assert');
 const path = require('path');
 const { execFileSync } = require('child_process');
 
 const script = path.join(__dirname, '..', '..', 'tools', 'doc', 'apilinks.js');
-const fixture = fixtures.path('apilinks', 'known1.js');
-const expected = {
-  'Known.classField': 'known1.js#L7',
-  'known1.createKnown': 'known1.js#L9'
-};
-const ret = execFileSync(
-  process.execPath, [script, fixture], { encoding: 'utf-8' }
-);
-const links = JSON.parse(ret);
-for (const [k, v] of Object.entries(expected)) {
-  assert.ok(k in links);
-  assert.ok(links[k].includes(v));
-  delete links[k];
-}
-assert.strictEqual(Object.keys(links).length, 0);
+
+const apilinks = fixtures.path('apilinks');
+fs.readdirSync(apilinks).forEach((fixture) => {
+  if (!fixture.endsWith('.js')) return;
+  const file = path.join(apilinks, fixture);
+
+  let expected = {};
+  eval(/(expected\s*=.*?);/s.exec(fs.readFileSync(file, 'utf8'))[1]);
+
+  const links = JSON.parse(execFileSync(
```

Comment:
IMO it's better to hold the returned output in a variable (where I used `ret`)
1. Single action per line - readability + help distinguish _what_ failed if this line fails
2. When doing interactive debugging, you can inspect this vale.
3. JS compiler could probably optimize the variale away